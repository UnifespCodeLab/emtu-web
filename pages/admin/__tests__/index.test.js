import { mount, createLocalVue } from '@vue/test-utils'
import Vuetify from 'vuetify'

import AdminPage from '~/pages/admin/index.vue'

describe('Pages / AdminPage', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should render the page', () => {
    const wrapper = mount(AdminPage, {
      localVue,
      vuetify
    })
    expect(wrapper.findAll('input').length).toBe(2)
    expect(wrapper.findComponent({ name: 'v-btn' }).exists()).toBe(true)
    expect(wrapper.element).toMatchSnapshot()
  })
})
