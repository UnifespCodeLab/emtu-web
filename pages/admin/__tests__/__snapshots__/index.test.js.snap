// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages / AdminPage should render the page 1`] = `
<div
  class="admin-page"
>
  <div
    class="admin-page__form"
  >
    <i
      aria-hidden="true"
      class="v-icon notranslate mb-6 mdi mdi-bus theme--light primary--text"
      style="font-size: 40px;"
    />
     
    <div
      class="admin-page__text text-center text-h6 mb-7"
    >
      Login usu√°rio Administrador
    </div>
     
    <div
      class="v-input theme--light v-text-field v-text-field--single-line v-text-field--solo v-text-field--enclosed"
    >
      <div
        class="v-input__control"
      >
        <div
          class="v-input__slot"
        >
          <div
            class="v-text-field__slot"
          >
            <label
              class="v-label theme--light"
              for="input-3"
              style="left: 0px; position: absolute;"
            >
              E-mail
            </label>
            <input
              id="input-3"
              type="text"
            />
          </div>
        </div>
        <div
          class="v-text-field__details"
        >
          <div
            class="v-messages theme--light"
          >
            <transition-group-stub
              class="v-messages__wrapper"
              name="message-transition"
              tag="div"
            />
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="v-input theme--light v-text-field v-text-field--single-line v-text-field--solo v-text-field--enclosed"
    >
      <div
        class="v-input__control"
      >
        <div
          class="v-input__slot"
        >
          <div
            class="v-text-field__slot"
          >
            <label
              class="v-label theme--light"
              for="input-6"
              style="left: 0px; position: absolute;"
            >
              Senha
            </label>
            <input
              id="input-6"
              type="password"
            />
          </div>
        </div>
        <div
          class="v-text-field__details"
        >
          <div
            class="v-messages theme--light"
          >
            <transition-group-stub
              class="v-messages__wrapper"
              name="message-transition"
              tag="div"
            />
          </div>
        </div>
      </div>
    </div>
     
    <button
      class="mb-6 v-btn v-btn--block v-btn--is-elevated v-btn--has-bg theme--light elevation-2 v-size--large primary"
      type="button"
    >
      <span
        class="v-btn__content"
      >
        
      enviar
    
      </span>
    </button>
     
    <div
      class="admin-page__text text-center text-caption color-gray"
    >
      Esqueceu sua senha?
    </div>
  </div>
</div>
`;
